src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringInterpolation.cs

```diff
@@ -35,6 +36,108 @@ private BoundExpression RewriteInterpolatedStringConversion(BoundConversion conv
             return result;
         }
 
+        private BoundNode RewriteToInterpolatedStringBuilderPattern(BoundInterpolatedString node)
+        {
+            Debug.Assert(node.InterpolationData is { Construction: not null });
+            // PROTOTYPE(interp-string): Support the general builder pattern that doesn't save to a local.
+            var resultSymbol = _factory.SynthesizedLocal(_compilation.GetSpecialType(SpecialType.System_String), node.Syntax);
+            var resultTemp = _factory.Local(resultSymbol);
+            var builderTempSymbol = _factory.InterpolatedStringBuilderLocal(node.InterpolationData.BuilderType, node.InterpolationData.ScopeOfContainingExpression, node.Syntax);
+            var builderTemp = _factory.Local(builderTempSymbol);
+
+            // PROTOTYPE(interp-string): Support optional out param for whether the builder was created successfully
+            // var builder = Construction(baseStringLength, numFormatHoles);
+            var builderConstruction = _factory.Assignment(builderTemp, MakeCallWithNoExplicitArgument(node.InterpolationData.Construction, node.Syntax, expression: null, assertParametersAreOptional: false));
```

Comment:
>MakeCallWithNoExplicitArgument [](start = 71, length = 30)

Why are we using "WithNoExplicitArgument" helper when we know that there are explicit arguments?